sapi/fpm/tests/status.inc

```diff
@@ -232,40 +232,36 @@ class Status
     {
         $pattern = "(# HELP phpfpm_up Could pool " . $fields['pool'] . " using a " . $fields['process manager'] . " PM on PHP-FPM be reached\?\n" .
             "# TYPE phpfpm_up gauge\n" .
-            "phpfpm_up 1\n" .
-            "# HELP phpfpm_start_since The number of seconds since FPM has started\.\n" .
-            "# TYPE phpfpm_start_since counter\n" .
-            "phpfpm_start_since " . $fields['start since'] . "\n" .
-            "# HELP phpfpm_accepted_connections The number of requests accepted by the pool\.\n" .
-            "# TYPE phpfpm_accepted_connections counter\n" .
-            "phpfpm_accepted_connections " . $fields['accepted conn'] . "\n" .
+            "phpfpm_up{pool=\"" . $fields['pool'] . "\"} 1\n" .
```

Comment:
I think test is failing due to the escaping here. Maybe it should be double escaped ? (not sure)
```suggestion
            "phpfpm_up{pool=\\\"" . $fields['pool'] . "\\\"} 1\n" .
```